src/app/components/playButton.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+
+export default class playButton extends Component {
+    constructor(props) {
+        super(props)
+        
+        this.state = {
+          percentage: 0,
+          play:false,
+          pause:false
+        }
+        
+        this.nextStep = this.nextStep.bind(this)
+      }
+      
+      nextStep() {
```

Comment:
can we use arrow functions to bind scope here?

```suggestion
      }
      
      nextStep = () => {
```